pychromecast/__init__.py

```diff
@@ -349,20 +349,19 @@ def reboot(self):
         """ Reboots the Chromecast. """
         reboot(self.host)
 
-    def volume_up(self):
-        """ Increment volume by 0.1 unless it is already maxed.
+    def volume_up(self, delta=0.1):
+        """ Increment volume by 0.1 (or delta) unless it is already maxed.
         Returns the new volume.
 
         """
-        volume = round(self.status.volume_level, 1)
-        return self.set_volume(volume + 0.1)
+        return self.set_volume(self.status.volume_level + abs(delta))
 
-    def volume_down(self):
-        """ Decrement the volume by 0.1 unless it is already 0.
+    def volume_down(self, delta=0.1):
+        """ Decrement the volume by 0.1 (or delta) unless it is already 0.
         Returns the new volume.
         """
-        volume = round(self.status.volume_level, 1)
-        return self.set_volume(volume - 0.1)
+        import sys
```

Comment:
Whoops.  That was from testing.  Sorry about that (too early in the morning, here). It's fixed: https://github.com/balloob/pychromecast/pull/104/commits/909e485e622bedfd181dae618690209a273218c6.